docs/provision.adoc

```diff
@@ -0,0 +1,46 @@
+[[custom-runtime-connector-service]]
+= Custom Runtime Connector
+
+Allows to expose an existing Custom Service to mobile application. Mobile perspective to any existing service could be added
+Custom runtime connector should create a configmap and then you can get the config back by running
+```bash
+oc plugin mobile create client <client-name> <clientType> <bundleID> --namespace <namespace>
```

Comment:
I don't think we need to add this command here since this is talking about getting the client config